docs/_sass/_colors.scss

```diff
@@ -2,16 +2,16 @@ $color-dark: #0B1B34;
 $color-gray1: #FAFAFA;
 $color-gray2: #ECECEC;
 $color-gray3: #C6C9CA;
-$color-green: #07d973;
+$color-green: #03D47C;
 $color-pink: #F68DFE;
-$color-white: #FFFFFF;
+$color-white: #E7ECE9;
 
 $color-light-grey-green: #C9D3C5;
 $color-gray-green: #8B9C8F;
-$color-dark-green: #1B5744;
-$color-darker-green: #002E22;
+$color-dark-green: #1A3D32;
+$color-darker-green: #07271F;
 $color-super-dark-green: #061B09;
```

Comment:
@shawnborton not sure on the name for `$color-light-grey-green: #C9D3C5;`

$color-green-icons: #8B9C8F;
$color-green-borders: #1A3D32;
$color-green-highlightBG: #07271F;
$color-green-appBG: #061B09;

if we decide to update color names

Reply:
@shawnborton do you prefer `color-appBG` or `color-green-appBG` for the pattern for these?

Reply:
I think I would separate colors from purposes, so we might do something like:

```
// Define dark theme color values
$color-green-appBG: #061B09;
$color-green-highlightBG: #07271F;
$color-green-borders: #1A3D32;
etc

// Dark theme colors as CSS vars
:root {
    --color-appBG: $color-green-appBG;
    --color-highlightBG: $color-green-highlightBG;
    --color-borders: $color-green-borders;
    etc
}
```

Then I would try to use CSS vars for values:

```
body {
    background-color: var(--color-appBG);
}
```

Then theme switching becomes super easy, we'd just add a modifer class to the body of the page, and define our new colors like so:

```
// Define light theme color values
$color-light-appBG: #FCFBF9;
$color-light-highlightBG: #F8F4F0;
$color-light-borders: #EBE6DF;
etc

// Dark theme colors as CSS vars
.lightmode {
    --color-appBG: $color-light-appBG;
    --color-highlightBG: $color-light-highlightBG;
    --color-borders: $color-light-borders;
    etc
}
```

So that this way the CSS vars stay the same throughout the app styles, but we just use modifier classes to change up the colors that define those CSS vars. Let me know if that makes sense!

Reply:
@shawnborton I see some color is definded but it doesn't use. Should we remove it?

Reply:
@shawnborton there are $color-dark, $color-gray1, $color-gray2, $color-gray3, $color-pink

Reply:
ah interesting, those are old colors from what the site looked like pre-rebrand. Yeah, we should remove those too.

Reply:
@shawnborton I try to use :root as above but it doesn't match color variable

Reply:
@grgia thoughts there?

I don't think we necessarily need to switch over to CSS variables here, that wasn't really part of what the original issue sought out to accomplish. But if we can make it work, that would be great.